packages/iproute2/build.sh

```diff
@@ -0,0 +1,15 @@
+TERMUX_PKG_HOMEPAGE=https://wiki.linuxfoundation.org/networking/iproute2
+TERMUX_PKG_DESCRIPTION="Utilities for controlling networking"
+TERMUX_PKG_LICENSE="GPL-2.0"
+TERMUX_PKG_MAINTAINER="@termux"
+TERMUX_PKG_VERSION=5.15.0
+TERMUX_PKG_SRCURL=https://www.kernel.org/pub/linux/utils/net/iproute2/iproute2-${TERMUX_PKG_VERSION}.tar.xz
+TERMUX_PKG_SHA256=38e3e4a5f9a7f5575c015027a10df097c149111eeb739993128e5b2b35b291ff
+TERMUX_PKG_DEPENDS="libandroid-glob, libandroid-support"
+TERMUX_PKG_BUILD_IN_SRC=true
+TERMUX_PKG_EXTRA_MAKE_ARGS="DESTDIR=${TERMUX_PREFIX} PREFIX="
```

Comment:
> PREFIX=

This will set wrong path values for CONFDIR, NETNS_RUN_DIR, NETNS_ETC_DIR, and few others that used by sources and manpages.

Additionally as following path is not configurable, it will require a patch:
```
misc/arpd.c
42:char *dbname = "/var/lib/arpd/arpd.db";
```

Reply:
Resolved. Thanks!